### PR TITLE
[NUI] Changes ModelView to Model

### PR DESCRIPTION
**Pull request by bshsqa**
Signed-off-by: seungho <sbsh.baek@samsung.com>

### Description of Change ###

This patch changes ModelView class to Model class.
And, unnecessary APIs have been removed.
And, default value of pivotPoint and parentOrigin of the Model are set to Center.

### API Changes ###

 - ACR:

### CODE REVIEW

**Review by hyunjushin — COMMENTED** [review]
unclear the meaning of model, model area.... please update it.

File: src/Tizen.NUI.Scene3D/src/public/Controls/Model.cs [hunk 1]
```diff
@@ -18,62 +18,65 @@
...
     /// }
     /// </code>
     // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
     [EditorBrowsable(EditorBrowsableState.Never)]
-    public class ModelView : View
+    public class Model : View
     {
-        private bool useSizeOfView = false;
-        private bool useCenterOfView = false;
+        private bool useCenterPivot = false;
```
Comment (hinohie) [inline src/Tizen.NUI.Scene3D/src/public/Controls/Model.cs:59]: Are you plan to use this value? (Something like.. Whether this model use Center/Center forcely, or use it's own Anchor/ParentOrigin)

If, or not If, I think we'd better remove this unused value

Comment (bshsqa) [reply src/Tizen.NUI.Scene3D/src/public/Controls/Model.cs]: Thank you I removed it.

**Review by Seoyeon2Kim — COMMENTED** [review]
Could you update the description on this PR?

**Review by Seoyeon2Kim — APPROVED** [review]
LGTM

**Review by dongsug-song — APPROVED** [review]
approve

**Pull request merged**